steering_controllers_library/include/steering_controllers_library/steering_odometry.hpp

```diff
@@ -237,6 +240,14 @@ class SteeringOdometry
    */
   double convert_twist_to_steering_angle(const double v_bx, const double omega_bz);
 
+  /**
+   * \brief Calculates angular velocity from the desired steering angle
+   * \param v_bx Desired linear velocity of the robot in x_b-axis direction
+   * \param phi Desired steering angle
```

Comment:
```suggestion
   * \brief Calculates angular velocity from the steering angle and linear velocity
   * \param v_bx Linear velocity of the robot in x_b-axis direction
   * \param phi Steering angle
```
can be used for odometry as well